### ISSUE TITLE
Fix incorrect message on iOS sign-in

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla/fxa-content-server#5858]
I've seen this myself after Sync Fest and a friend just emailed me the screenshot. Please let's fix this one ASAP.
<img width="895" alt="image" src="https://user-images.githubusercontent.com/68213/35461692-e5e4fdd0-02b6-11e8-824d-0e62b58cc276.png">

**Comment by ryanfeeley** [comment]
Related: https://github.co/mozilla/fxa-content-server/issues/5434

**Comment by rfk** [comment]
@ryanfeeley it's not clear to me whether https://github.com/mozilla/fxa-content-server/issues/5434 is a dupe of this, or is slightly different; can you please clarify?

**Comment by ryanfeeley** [comment]
@rfk This issue puts the solution up top, which got lost in the length of the previous issue.

**Comment by davismtl** [comment]
@ryanfeeley I'm not sure I understand the proposal though. Feels weird to display CAD on mobile after sign-in and especially in this proposed design where the user is on iOS and we're displaying an Android install button. I don't see this converting in any way.

Can we make better use of this screen? Should we show users how to Send Tab?

**Comment by rfk** [comment]
> Should we show users how to Send Tab?

I like the idea, but I propose that this is out of scope for Q1, and we should focus on just getting this to show a sensible state rather than looking like we can't tell whether the user is signed in or not.

**Comment by davismtl** [comment]
> I like the idea, but I propose that this is out of scope for Q1, and we should focus on just getting this to show a sensible state rather than looking like we can't tell whether the user is signed in or not.

So showing an Android install button to iOS users is a really bad idea.

Whether it be send tab or something else, we can make better use of the image and copy and this screen.

**Comment by vbudhram** [comment]
> getting this to show a sensible state

Would a good stop gap fix for this be to update the text to "Sit adding devices? Sign in to Firefox on another device to complete setup?" and not display any App Store buttons?

**Comment by davismtl** [comment]
That seems like the easiest but still seems weird and as useless as a green checkmark.

@ryanfeeley Feeley was exploring copy yesterday by testing other apps. He shared this example:

![img_5201](https://user-images.githubusercontent.com/19537086/35589742-aee89b88-05b9-11e8-80eb-c1ff1e110c08.PNG)
![img_5200](https://user-images.githubusercontent.com/19537086/35589743-aefc0ff6-05b9-11e8-98b4-712a9f3788c4.PNG)

Seems to me that better copy should be easy without much overhead for us.

**Comment by ryanfeeley** [comment]
Perfect is the enemy of the good.

@davismtl Remember that what we are showing today is baffling to users. Let's remedy that right now for all iOS users.

I don't think it makes sense to show the App Store because the user is in Firefox on iOS.

We know that they are connected because the green message says "This Firefox is connected". This link will also open in Firefox thanks to universal links.

I chose the Play Store icon specifically because the user may have an Android tablet.

**Comment by davismtl** [comment]
> Perfect is the enemy of the good.
Agreed

> I chose the Play Store icon specifically because the user may have an Android tablet.
Yes but if they click this on their iPhone, it's useless. (we are on iOS here)

So either, we show both buttons or no buttons, but let's not put an Android install button on iOS.

**Comment by ryanfeeley** [comment]
@davismtl I have an Android tablet and use an iPhone. This is not uncommon.

**Comment by davismtl** [comment]
@ryanfeeley 

I'm not saying it is uncommon but it's just not the majority so the UX is irrelevant for most people. I'm questioning putting an Android install button on an iOS device and if we can come up with anything better that is minimal work. 

Can the copy at least call out Android specifically? (e.g. Have you added your Android devices?)

Since this UX is not common, I'm not sure what the user journey  is suppose to be. Is the below what you have in mind?
1) click on app store button, 
2) sign in to Google Account 
3) install Android app via mobile web interface

I would personally consider it a bug if a website was showing me an Android install button while on my iPhone. So, to me, your proposed change would be no better than the original screen. It's like replacing a bug for another bug. Let's make this more relevant for the majority of our users.

**Comment by ryanfeeley** [comment]
It’s not likely anyone would click on it, but it lets users know we are available on Android. Please unblock ASAP and I promise I’ll follow up with something great for the multi-device moment.

**Comment by davismtl** [comment]
Per meeting with @ryanfeeley and @vbudhram this morning, we will go with the proposed design but add both app store buttons.

We realized that the same screen is displayed when you confirm both a login and a registration in iOS.
